src/screens/FeedScreen/RenderList.js

```diff
@@ -146,6 +149,7 @@ const RenderListFeed = (props) => {
             messageContainerStyle={{paddingHorizontal: 10}}
             topics={item?.topics}
             contentHeight={contentLinkHeight()}
+            isHaveComment={isHaveComment}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isHaveComment` is passed to the `ContentLink` component. However, the prop name `isHaveComment` is grammatically incorrect and can be confusing. Consider renaming it to `hasComment` for better readability.
```diff
- isHaveComment={isHaveComment}
+ hasComment={hasComment}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
semua diganti aja ya sekalian berarti

Reply:
updated gan @asaadam